src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -620,7 +622,18 @@ internal void Save(string filePath, bool isBackup = false, EngineController engi
                 var jo = AddViewBlockToJSON(json_parsed);
 
                 // Stage 3: Save
-                File.WriteAllText(filePath, jo.ToString());
+                string saveContent;
```

Comment:
do you think this this be optionally controlled?

Reply:
Sorry can you elaborate?

Reply:
I mean, when saving, it might be useful, if you could disable this guid update.

I see that this method is internal, so it's not like the behavior can be changed by extension authors anyway, but it might be worth considering what use cases this change in behavior might affect.